src/pages/Analytics/TEI/AnalyticsTEI.js

```diff
@@ -28,12 +33,11 @@ const AnalyticsTEI = () => {
     }, [hasAuthority])
 
     useEffect(() => {
-        if (analytics) {
-            const settings = analytics.analytics.tei
-            setInitialValues(settings)
-            setAnalyticSettings(settings)
+        if (tei) {
+            setInitialValues(tei)
```

Comment:
Is `initialValues` still required or can `tei` be used directly instead? It seems like `initialValues` and `tei` are kept in sync?